app/models/vbms_uploaded_document.rb

```diff
@@ -5,6 +5,10 @@ class VbmsUploadedDocument < CaseflowRecord
 
   attribute :file, :string
 
+  # rubocop:disable Layout/LineLength
+  scope :successfully_uploaded, -> { where(error: nil).where.not(uploaded_to_vbms_at: nil, attempted_at: nil, processed_at: nil) }
+  # rubocop:enable Layout/LineLength
+
```

Comment:
```suggestion
  scope :successfully_uploaded, lambda {
    where(error: nil).where.not(uploaded_to_vbms_at: nil, attempted_at: nil, processed_at: nil)
  }

```

Using the lambda method allows for multiline lambdas to be defined without rubocop complaining.

Reply:
Implemented